scanner/src/components/Divider/index.svelte

```diff
@@ -0,0 +1,8 @@
+<div class="divider" />
+
+<style lang="scss">
+  .divider {
+    border-bottom: 1px solid #20303e;
```

Comment:
looks like this color should be an scss variable
![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/6092ad7b-70d4-4f04-953b-47b8a490efdb)